superset-frontend/src/views/CRUD/utils.tsx

```diff
@@ -28,14 +28,24 @@ const createFetchResourceMethod = (method: string) => (
   resource: string,
   relation: string,
   handleError: (error: Response) => void,
-) => async (filterValue = '', pageIndex?: number, pageSize?: number) => {
+) => async (
+  filterValue = '',
+  pageIndex?: number,
+  pageSize?: number,
+  sortValue?: string,
+  sortDirection = 'asc',
```

Comment:
let's scrap this. I was wrong, the `related|distinct` endpoints don't take `order_column` and `order_direction` params (https://github.com/apache/incubator-superset/blob/10a54b52ac522ff41bd45d17c5deb417dfa8a359/superset/views/base_api.py#L46) so this is doing nothing

Reply:
I think the "bugs" were regarding the chart's `viz_type` and `datasource` columns.